src/main/java/hudson/plugins/jira/JiraCreateIssueNotifier.java

```diff
@@ -463,10 +469,32 @@ public FormValidation doCheckProjectKey(@QueryParameter String value) throws IOE
             return FormValidation.ok();
         }
 
-        public ListBoxModel doFillPriorityIdItems() {
+        protected List<JiraSite> getJiraSites(Item item){
```

Comment:
This whole block below looks universal and we already have similar methods in JiraSite, so that should be moved together with the rest imo.